app-deployment-manager/test/deployment_update/deployment_update_test.go

```diff
@@ -0,0 +1,88 @@
+// SPDX-FileCopyrightText: (C) 2025-present Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package deployment_update
+
+import (
+	"net/http"
+
+	"github.com/open-edge-platform/app-orch-deployment/app-deployment-manager/api/nbi/v2/pkg/restClient"
+	deploymentutils "github.com/open-edge-platform/app-orch-deployment/test-common-utils/pkg/deployment"
+)
+
+func (s *TestSuite) TestUpdateDeploymentValidParams() {
+	s.T().Parallel()
+	testName := "TestUpdateDeploymentValidParams"
+	deploymentReq := deploymentutils.StartDeploymentRequest{
+		AdmClient:         s.AdmClient,
+		DpPackageName:     deploymentutils.AppNginx,
+		DeploymentType:    deploymentutils.DeploymentTypeTargeted,
+		DeploymentTimeout: deploymentutils.DeploymentTimeout,
+		DeleteTimeout:     deploymentutils.DeleteTimeout,
+		TestName:          testName,
+	}
+	deployID, code, err := deploymentutils.StartDeployment(deploymentReq)
+	s.Equal(http.StatusOK, code)
+	s.NoError(err, "Failed to create '"+deploymentutils.AppNginx+"-"+deploymentutils.DeploymentTypeTargeted+"' deployment")
+
+	deployment, code, err := deploymentutils.GetDeployment(s.AdmClient, deployID)
+	s.Equal(http.StatusOK, code, "Expected HTTP status 200 for getting deployment details")
+	s.NoError(err, "Failed to get deployment details")
+
+	// TODO: some modification the deployment object to update?
```

Comment:
I think we should not postpone this as TODO and we should try to find a valid update. for example you can use wordpress to upgrade the app version and this is the main goal of this test to see if Upgrade works or not.